app/views/components/gds/exportsInsetText.scala.html

```diff
@@ -0,0 +1,31 @@
+@*
```

Comment:
Do you really need a new component? Do you expect that it will also be used in other templates?

Reply:
Yeah, I think Liz has plans to use them in other pages too